### PR TITLE
Ben/docu 1056

### PR DESCRIPTION
**Pull request by benradstone**

### CODE REVIEW

File: docs/configuration/config-properties/component-files.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+id: component-files
+title: Component files
+sidebar_label: Component files
+---
+
+You can configure LiveData Plane’s component applications using the following files. Take care when making any configuration changes on your clusters.
```
Comment (stepgerner) [inline docs/configuration/config-properties/component-files.md:7]: Can you maybe elaborate on "take care"/why you're suggesting this? If it's abundantly obvious to our target audience don't worry though.

File: docs/configuration/config-properties/component-files.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+- `/etc/wandisco/fusion/server/application.properties` - contains LiveData Plane Server properties
+- `/etc/wandisco/fusion/ihc/server/{distro}/{version string}.ihc` - contains all the IHC server properties
+- `config.properties` - contains all the LiveData Plane Client properties. Using this file you can make updates to LiveData Plane Client properties without editing the `core-site.xml`, which would require a restart of Hadoop services.
```
Comment (stepgerner) [inline docs/configuration/config-properties/component-files.md:11]: Minor nitpick, this bulletpoint ends with a period while the others don't. May be good to make them consistent.

File: docs/configuration/config-properties/ihc-server.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
+---
+id: ihc.server
+title: Inter-Hadoop Communication Server
+sidebar_label: IHC Server
+---
+
+The Inter-Hadoop Communication (IHC) Server is configured from a single file located at `/etc/wandisco/fusion/ihc/server/{distro}/{version string}.ihc`.
+
+All properties in this file are checked at startup. If set options are given, the default is highlighted in bold, e.g. True/**false**.
```
Comment (stepgerner) [inline docs/configuration/config-properties/ihc-server.md:9]: Is "if set options are given" the right phrase here? Sorry, but I don't understand what this means. Does it just mean "In properties where you must select from pre-existing options, the default value is highlighted in bold"?

File: docs/configuration/config-properties/ihc-server.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+
+### `ihc.ssl.key.alias`
+
+Alias of private key / certificate chain of the IHC server used to encrypt communications.
+
+- Alias of a keystore entry
+
+### `ihc.ssl.key.password`
+
+Encrypted password of private key entry in keystore. Can be encrypted using encrypt-password.sh.
```
Comment (stepgerner) [inline docs/configuration/config-properties/ihc-server.md:57]: For consistency, should all of these be like "The encrypted password of the private key entry in the keystore"?

File: docs/configuration/config-properties/ihc-server.md [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+
+### `ihc.ssl.keystore`
+
+Local filesystem path of key store containing key entry.
```
Comment (stepgerner) [inline docs/configuration/config-properties/ihc-server.md:61]: Similar to the above, maybe "The local filesystem of the key store..." I won't repeat this comment for each one though

File: docs/operation/consistency-check.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
+---
+id: consistency-check
+title: Run a Consistency Check
+sidebar_label: Consistency Check
+---
+
+Once you have [created a replication rule](./create-rule.md), run a consistency check to compare the contents between all zones.
+
+1. On the **Rules** table, click to **View rule**.
```
Comment (stepgerner) [inline docs/operation/consistency-check.md:9]: Suggest "Click View rule" instead of "Click to"

File: docs/operation/consistency-check.md [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+
+2. On the rule page, **start consistency check** and wait for the **Consistency status** to update. The more objects contained within the path, the longer it will take to complete the check.
```
Comment (stepgerner) [inline docs/operation/consistency-check.md:11]: Suggest "On the rule page, click **start consistency check**"

File: docs/operation/consistency-check.md [hunk 8]
```diff
@@ -0,0 +1,15 @@
...
+
+   The **Consistency Status** will determine the next steps:
+   * **Consistent** - no further action is required.
```
Comment (stepgerner) [inline docs/operation/consistency-check.md:14]: Might be worth a small elaboration, like "**Consistent** - the contents between all zones are consistent. No further action is required."

File: docs/operation/consistency-check.md [hunk 9]
```diff
@@ -0,0 +1,15 @@
...
+   * **Inconsistent** - consider [starting a migration](./migration.md).
```
Comment (stepgerner) [inline docs/operation/consistency-check.md:15]: "**Inconsistent** - discrepancies have been found between files on your source and the target. Consider starting a migration to update the target with the changes." (But with the links intact)

File: docs/quickstarts/preparation/get-started-trial.md [hunk 10]
```diff
@@ -1,14 +1,18 @@
...
+
+Our pre-configured environments use [Docker](https://docs.docker.com/get-started/overview/) to run a containerized deployment of LiveData Plane.
+
 ## Before you begin
 
-Select your [pre-configured environment](../installation/installation-quickstarts.md) before running through these instructions.
+Decide on your [pre-configured environment](../installation/installation-quickstarts.md) before running through these instructions.
 
-All Dockerized LiveData Plane environments require a host machine that meets minimum specifications, and each pre-configured environment may vary in the minimum specification needed.
+All Dockerized LiveData Plane environments require a host machine that meets minimum specifications, and each environment may vary in the minimum specification needed.
```
Comment (stepgerner) [inline docs/quickstarts/preparation/get-started-trial.md:15]: Is there a link to minimum specifications you can put here just for user convenience?

**Review by lucasdigital — APPROVED** [review]
LGTM. The best test will be using the new site.

**Review by stepgerner — CHANGES_REQUESTED** [review]
Some comments/nitpicks. Reads great overall, really clicks for me.

**Pull request merged**